ui/src/app/shared/model/JobStatus.ts

```diff
@@ -18,5 +18,6 @@ export enum JobStatus {
     Aborting = <any> 'Aborting',
     Failed = <any> 'Failed',
     Succeeded = <any> 'Succeeded',
-    Aborted = <any> 'Aborted'
+    Aborted = <any> 'Aborted',
+    OnHold = <any> 'On Hold'
```

Comment:
Why does `OnHold` not have a space here like the model definition does?

Reply:
I assume it's a swagger codgen thing.

Reply:
ok!